layers/+emacs/org/README.org

```diff
@@ -423,6 +424,18 @@ More information about org-roam package (including manual) can be found at [[htt
 To install support for [[https://github.com/org-roam/org-roam-server][org-roam-server]] set the variable =org-enable-roam-server=
 to =t=.
 
+*** Org-roam-protocol support
+To install support for [[https://www.orgroam.com/manual.html#Roam-Protocol][Org Roam Protocol]] set the variable
```

Comment:
From the manual:
> To enable Org-roam’s protocol extensions

There's nothing to be installed. `org-roam-protocol` is included in `org-roam`.

Reply:
Thanks for pointing that one out. I think I copied that section from one of the other sections, and must have forgotten to change that. I'll push a fix later today.